firebase-vertexai/src/main/kotlin/com/google/firebase/vertexai/FirebaseVertexAI.kt

```diff
@@ -55,9 +58,13 @@ class FirebaseVertexAI(
     tools: List<Tool>? = null,
     toolConfig: ToolConfig? = null,
     systemInstruction: Content? = null,
-  ) =
-    GenerativeModel(
-      "projects/${firebaseApp.options.projectId}/locations/${LOCATION}/publishers/google/models/${modelName}",
+    location: String = "us-central1"
+  ): GenerativeModel {
+    if (location.isEmpty() || location.trim().isEmpty() || location.contains("/")) {
```

Comment:
No need for the initial `isEmpty()`

```suggestion
    if (location.trim().isEmpty() || location.contains("/")) {
```